subsys/net/ip/net_shell.c

```diff
@@ -2799,26 +2800,32 @@ static int cmd_net_ping(const struct shell *shell, size_t argc, char *argv[])
 
 	shell_for_ping = shell;
 
-	if (IS_ENABLED(CONFIG_NET_IPV6)) {
-		ret = ping_ipv6(shell, host);
-		if (!ret) {
-			goto wait_reply;
-		} else if (ret == -EIO) {
-			PR_WARNING("Cannot send IPv6 ping\n");
-			return -ENOEXEC;
-		}
+	if (argv[2]) {
```

Comment:
I'm not sure if argv[i] are set to NULL by default, if not, we need check argc as well. 
if ((argc > 2) && argv[2])

Reply:
Just refer to argv[1]

Reply:
Just refer to argv[1]

Reply:
Thanks @easonxiang.
Just refer to argv [1] above.

Well.We might follow another PR Jukkar memtioned.